packages/web/src/components/apps/header.tsx

```diff
@@ -170,25 +171,46 @@ export default function EditorHeader(props: PropsType) {
               </TooltipProvider>
             ) : null}
             {props.tab === 'preview' && previewStatus !== 'stopped' ? (
-              <TooltipProvider>
-                <Tooltip>
-                  <TooltipTrigger asChild>
-                    <Button
-                      variant="icon"
-                      size="icon"
-                      onClick={() => {
-                        stopPreview();
-                        props.onChangeTab('code');
-                      }}
-                      className="active:translate-y-0"
-                      disabled={previewStatus === 'booting' || previewStatus === 'connecting'}
-                    >
-                      <StopCircleIcon size={18} />
-                    </Button>
-                  </TooltipTrigger>
-                  <TooltipContent>Stop dev server</TooltipContent>
-                </Tooltip>
-              </TooltipProvider>
+              <>
+                {url && (
+                  <TooltipProvider>
+                    <Tooltip>
+                      <TooltipTrigger asChild>
+                        <Button
+                          variant="icon"
+                          size="icon"
+                          onClick={() => window.open(url as string, '_blank')}
+                          className="active:translate-y-0"
+                          disabled={previewStatus === 'booting' || previewStatus === 'connecting'}
```

Comment:
nit but probs better to do `previewStatus !== 'running'`, especially since at some point stop may keep the preview stopped (rather than client automatically booting it back up again, which is currently awkward behavior given that we have a stop button that doesn't really stop it).